src/clientdynamicinfo.h

```diff
@@ -33,11 +33,13 @@ struct ClientDynamicInfo
 	f32 real_gui_scaling;
 	f32 real_hud_scaling;
 	v2f32 max_fs_size;
+	bool touch_controls;
 
 	bool equal(const ClientDynamicInfo &other) const {
```

Comment:
unrelated but who wrote this and didn't use `operator==`

Reply:
🤷